backend/development.ini.sample

```diff
@@ -101,6 +101,9 @@ auth_type = internal
 # ldap_group_enabled = False
 # User auth token validity in seconds (used to interfaces like web calendars)
 user.auth_token.validity = 604800
+# user reset_password token validity (default to 600 -> 5 minutes)
```

Comment:
what is the unit ?! 600 seconds = 10 minutes, not 5

Reply:
just an error. i fix that.